templates/Web/Pages/Angular.MasterDetail/src/app/app-shell/AngularMasterDetail/AngularMasterDetail-routing.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+import { MasterDetailComponent } from './master-detail.component';
+
```

Comment:
Remove one line

Reply:
The reason there is line to seperate import built in package and import custom packages